client/components/OrderDetails.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+class OrderDetails extends Component {
+  render() {
+    const { orders } = this.props;
+    const orderId = +this.props.match.params.orderId;
```

Comment:
I find this a weird practice and not very clear.  Just personal preference.  Can leave if want to.
```suggestion
    const orderId = Number(this.props.match.params.orderId);
```